chia/wallet/wallet_blockchain.py

```diff
@@ -95,10 +95,7 @@ async def create(
         self.lock = asyncio.Lock()
         self.coin_store = coin_store
         self.tx_store = tx_store
-        cpu_count = multiprocessing.cpu_count()
-        if cpu_count > 61:
-            cpu_count = 61  # Windows Server 2016 has an issue https://bugs.python.org/issue26903
-        num_workers = max(cpu_count - 2, 1)
```

Comment:
You deleted the num_workers here, so the system breaks. Might be that this is the quantity you want to compute in your function.

Reply:
Thanks! Sorry for this :sweat_smile:  -  I was hoping to run CI before review happens and catches ugly mistakes like these